src/main/java/com/syntaxphoenix/spigot/smoothtimber/listener/BlockBreakListener.java

```diff
@@ -82,76 +77,18 @@ public void onBlockBreak(final BlockBreakEvent event) {
             }
             event.setCancelled(true);
             CooldownHelper.trigger(player);
-            Bukkit.getScheduler().runTaskAsynchronously(PluginUtils.MAIN, (Runnable) () -> {
```

Comment:
Instead of moving those runnables into a new class that extends another class than the original it I would appreciate it if you would simply move this into another function inside this class and then reference it using `this::<function name>`

Reply:
WIth the approach that I mentioned in the PR comment above there isn't even a need to move this at all